SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslationManager.cpp

```diff
@@ -71,6 +87,18 @@ void SpatialVirtualWorkerTranslationManager::SpawnPartitionEntitiesForVirtualWor
 	}
 }
 
+void SpatialVirtualWorkerTranslationManager::ReclaimPartitionEntities()
+{
+	Partitions.Empty();
+	for (VirtualWorkerId VirtualWorkerId : VirtualWorkersToAssign)
+	{
+		Worker_PartitionId PartitionId = Translator->GetPartitionEntityForVirtualWorker(VirtualWorkerId);
+		check(PartitionId != SpatialConstants::INVALID_ENTITY_ID);
+		Partitions.Emplace(PartitionInfo{ PartitionId, VirtualWorkerId, SpatialConstants::INVALID_ENTITY_ID });
```

Comment:
Have u run clang-format? Might've expected it to expand these braces a bit

Reply:
Yep, just the way it likes it.